scss/EditModel/EditDataEntryForm.scss

```diff
@@ -133,6 +134,3 @@
   }
 }
 
-.expand {
-  @extend .item.active;
-}
```

Comment:
This syntax was no longer considered valid, so I changed it to the above, which is functionally equivalent.

Reply:
This is only functionally when `.expand` is inside `.item`, which does not have to be the case in the old code.
If you know that this will always be the case, then the change is OK, otherwise it has to to sth like this:

```scss
.item.active, .expand {
  a {
    cursor: pointer;
    &:hover {
      text-decoration: underline;
    }
  }
}

.item.inactive {
  a {
    color: inherit;
    opacity: 0.5;
    cursor: default;
  }
}
```